gdf.js

```diff
@@ -573,6 +577,7 @@ export class GDF {
     if (this.nestjs) runtime = 'NestJS'
     if (this.gatsby) runtime = 'Gatsby'
     if (this.adonisjs) runtime = 'AdonisJS'
+    if (this.vite) runtime = 'Vite'
```

Comment:
Move this above the other frameworks.  Remix, for example, is adding Vite support: https://remix.run/docs/en/main/future/vite.  So: if vite is in use and no other framework, it will report Vite, but if Vite is used in addition to another framework, that other framework will be the one reported.

Reply:
Good catch! Will update